internal/runners/checkout/checkout.go

```diff
@@ -83,6 +88,33 @@ func (u *Checkout) Run(params *Params) (rerr error) {
 		return locale.WrapError(err, "err_project_frompath")
 	}
 
+	if params.LibcVersion != "" {
```

Comment:
Nit: It may be worth putting this all in a separate function and calling it from here to keep the function smaller.